CHANGELOG.md

```diff
@@ -3,13 +3,84 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.7.0
+
+_Mar 24, 2022_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- ✏ Add a new editing API with better support for server-side persistence and validation (#3963) @m4theushw
+
+  The new API is only accessible by enabling the following flag:
```

Comment:
```suggestion
  The new API is stable, but to avoid any breaking changes or conflicts with the old API, it is accessible by enabling the following flag:
```

Reply:
Iterating on @joserodolfofreitas's suggestion.

```suggestion
  The new API is stable, but to avoid any breaking changes or conflicts with the old API, you must add the following flag to access it:
```